src/utils/mapping/section/death/mapping-utils.ts

```diff
@@ -71,6 +71,24 @@ export function getSectionMapping(mappingId: string): ISectionMapping {
           operation: 'getDeathRegistrationSectionTransformer'
         }
       }
+    case 'mother':
+      return {
+        query: {
+          operation: 'emptyMotherSectionTransformer'
+        }
+      }
+    case 'father':
+      return {
+        query: {
+          operation: 'emptyFatherSectionTransformer'
+        }
+      }
+    case 'spouse':
+      return {
+        query: {
+          operation: 'getSpouseSectionTransformer'
```

Comment:
Should we name this `emptySpouseSectionTransformer`? Not a big deal at all – just thinking after we release 1.3, the name might live on for quite some time

Reply:
Lets go with emptySpouseSectionTransformer in case we need to do anything different between spouse and father